actions/web_actions.js

```diff
@@ -1,30 +1,6 @@
-import {
-    ADD_TAGS_TO_WEB_IMAGE,
-    LOAD_MORE_WEB_IMAGES,
-    INCREMENT_WEB_IMAGES_UPLOADED,
-    REMOVE_TAG_FROM_WEB_IMAGE,
-    REMOVE_WEB_IMAGE,
-    WEB_IMAGES,
-    URL
-} from './types';
+import { ADD_IMAGE, DELETE_IMAGE, LOAD_MORE_WEB_IMAGES, URL } from './types';
```

Comment:
Please use multiline

Reply:
This done automatically based on .prettierrc.js config... 
I will configure it to make this multiline and create a new PR for that...